examples/hooks/App.css

```diff
@@ -54,3 +54,23 @@ body {
 .ais-CurrentRefinements-label {
   text-transform: capitalize;
 }
+
+.ais-PoweredBy {
+  margin-top: 0.5rem;
+}
+
+.ais-PoweredBy-text {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
+  white-space: nowrap;
+  border-width: 0;
+}
```

Comment:
This visually hides the element while keeping it visible for screen readers.

Reply:
shouldn't we have the test be inside the svg as a title or something?

Reply:
I used the same HTML as the [current component](https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch-dom/src/components/PoweredBy.js). In a previous PR we agreed not to drift for now.

Reply:
ah yes, but React InstantSearch is the only one which diverts from the spec, InstantSearch.js has the right dom: https://github.com/algolia/instantsearch.js/blob/master/src/components/PoweredBy/PoweredBy.tsx

see also #3044

Reply:
:ah: Gotcha, addressed in f325e9c8.

I didn't add the `theme` option since it's a DOM option, it doesn't make sense in a Hook.

We can add a full dark theme to the demo and toggle the `ais-PoweredBy--dark`/`ais-PoweredBy--light` class, but let's maybe finish the demo with all the Hooks first, and tackle this with other UI topics such as responsiveness, wdyt?